sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -519,33 +521,46 @@ def process(
       self.assertEqual(1, len(counters))
       self.assertEqual(counters[0].committed, len(''.join(data)))
 
-  def _run_sdf_wrapper_pipeline(self, source, expected_value):
+  def _run_sdf_wrapper_pipeline(self, source_ptransform, expected_value):
     with self.create_pipeline() as p:
       from apache_beam.options.pipeline_options import DebugOptions
       experiments = (p._options.view_as(DebugOptions).experiments or [])
 
       # Setup experiment option to enable using SDFBoundedSourceWrapper
-      if not 'use_sdf_bounded_source' in experiments:
+      if 'use_sdf_bounded_source' not in experiments:
         experiments.append('use_sdf_bounded_source')
-
       p._options.view_as(DebugOptions).experiments = experiments
 
       actual = (
-          p | beam.io.Read(source)
+          p | source_ptransform
       )
-    assert_that(actual, equal_to(expected_value))
+
+      assert_that(actual, equal_to(expected_value))
 
   @mock.patch('apache_beam.io.iobase._SDFBoundedSourceWrapper.expand')
   def test_sdf_wrapper_overrides_read(self, sdf_wrapper_mock_expand):
     def _fake_wrapper_expand(pbegin):
+      assert isinstance(pbegin, pvalue.PBegin)
+      return (pbegin
+              | beam.Create(['1']))
+
+    sdf_wrapper_mock_expand.side_effect = _fake_wrapper_expand
+    self._run_sdf_wrapper_pipeline(beam.io.Read(RangeSource(0, 4)), ['1'])
```

Comment:
This test looks really wrong... if we're going to use a mock perhaps have the element be 'mocked_data' or something obvious like that.

Reply:
Thanks for helping review this. I agree that the data should be `mockd_data`. Beside this, anything else looks wrong?

Reply:
No, just that.